### ISSUE TITLE
Release planning for v5.2.3 and beyond

### ISSUE THREAD

**Issue by Jermolene** [Jermolene/TiddlyWiki5#6765]
The work in #6666 to implement parameterised transclusion has proved to be substantial, and is not done yet after 2.5 months and more than 100 individual file modifications. It  promises to be the biggest simple improvement yet to TiddlyWiki's basic primitives, ushering in a new era with radically redrawn options for architecting reusable components within TiddlyWiki.

So, I propose:

* To hold #6666 over until the next release and give it a brand new version bump to v5.3.0
* In the meantime, to release v5.2.3 as soon as it is ready, hopefully by the end of the month

Please let me know if there are any further open PRs that you'd like considered for v5.2.3:

* [ ] #6626 
* [ ] #6588 
* [x] #6740 
* [x] #6671 
* [ ] #6528 
* [x] #6778 
* [x] #6771

Tasks for the release of v5.2.3:

* [x] Update release note with details of most recent commits
* [x] New release banner competition

**Comment by rmunn** [comment]
I have a mostly-completed work-in-progress on https://github.com/Jermolene/TiddlyWiki5/issues/6551 that stalled because I wasn't confident in the wording of the documentation I'd written. But the code is all there, and I believe it has tests as well. And if documentation wording is poor, that can be fixed without needing to push a new release. So I'll pick up my WIP and make a PR of it in the next few days.

**Comment by rmunn** [comment]
https://github.com/Jermolene/TiddlyWiki5/pull/6771, adding the `insertafter` operator, is now ready to be reviewed for inclusion in the 5.2.3 release.

**Comment by TiddlyTweeter** [comment]
I'm often off the main plan. But where else would I say this?

I just to want to say the whole TW thing is wonderful.
It is now ultra obvious TW is, basically, the way to interactively make a web page.

It's RECURSION is the best of anything I ever seen.

My point: Can we have a next version **dedicated to promoting TW?**

**Comment by Jermolene** [comment]
> I have a mostly-completed work-in-progress on #6551 that stalled because I wasn't confident in the wording of the documentation I'd written. But the code is all there, and I believe it has tests as well. And if documentation wording is poor, that can be fixed without needing to push a new release. So I'll pick up my WIP and make a PR of it in the next few days.

Thanks, please do.

> #6771, adding the `insertafter` operator, is now ready to be reviewed for inclusion in the 5.2.3 release.

Merged.

**Comment by EvidentlyCube** [comment]
@Jermolene 
Whoops, apparently I either can't copy or use Github interface. I meant to link to #6735, sorry!

**Comment by rmunn** [comment]
Just noticed a small grammatical mistake in the 5.2.3 release notes, and created https://github.com/Jermolene/TiddlyWiki5/pull/6800 to fix it.

**Comment by rmunn** [comment]
Also spotted an improvement I could make to the `insertafter` filter operator documentation, and created https://github.com/Jermolene/TiddlyWiki5/pull/6801 for that. That PR is against `master` because I'm figuring it can get merged in before the release. If it doesn't get merged in until after the release, I'll close that PR and open a new PR against the `tiddlywiki-com` branch instead. (And the same goes for #6800 as well).

**Comment by saqimtiaz** [comment]
@Jermolene I would like #6806 to be considered, as a self-contained and useful extension to filter affordances.

**Comment by Jermolene** [comment]
> @Jermolene I would like #6806 to be considered, as a self-contained and useful extension to filter affordances.

Thanks @saqimtiaz, that's merged.

**Comment by saqimtiaz** [comment]
#6809 Image lazy loading support for the $image widget. Works really well for external images.

**Comment by rmunn** [comment]
https://github.com/Jermolene/TiddlyWiki5/pull/6824 is literally a one-character chage, as a follow-up to #6801. In #6801 copied some text from the `insertbefore` operator docs to the `insertafter` docs, where a parameter named `B` in insertbefore is named `A` in insertafter. So I changed most cases of `<<.place B>>` to `<<.place A>>`, but I just now noticed that I missed one. #6824 fixes that one incorrectly-named parameter.
